### PR TITLE
[#161127] Docs tab for users

### PR DESCRIPTION
**Pull request by jossim**
# Release Notes

This adds a document tab for users that admins can use to upload files associated with a user.

This PR is a cherry-picked of https://github.com/Oregon-State/nucore-osu/pull/238

# Screenshot

Screen Shot 2023-01-09 at 4 53 31 PM

### CODE REVIEW

**Review by giladshanan — COMMENTED** [review]
@jossim did you run the migration locally?  I think the specs are failing because the timestamp version in `schema.rb` should match the filename of the most recent migration file:
```
ActiveRecord::Schema.define(version: 2022_12_12_231558) do
```

File: db/schema.rb [hunk 1]
```diff
@@ -213,7 +213,7 @@
     t.boolean "is_active", null: false
     t.integer "created_by", null: false
     t.datetime "created_at", null: false
-    t.integer "revenue_account", null: false
+    t.string "revenue_account", null: false
```
Comment (giladshanan) [inline db/schema.rb:216]: @jossim this doesn't look right.  I think your local db might be in a funky state. See https://github.com/tablexi/nucore-open/pull/3137

**Review by giladshanan — CHANGES_REQUESTED** [review]

File: app/views/admin/shared/_tabnav_users.html.haml [hunk 2]
```diff
@@ -26,4 +26,6 @@
   - if ResearchSafetyCertificate.any?
     = tab text("views.user_research_safety_certifications.tab"), facility_user_user_research_safety_certifications_path(current_facility, @user), secondary_tab == "user_certificates"
 
+  = tab "Docs", facility_user_user_file_uploads_path(current_facility, @user), secondary_tab == "docs"
```
Comment (giladshanan) [inline app/views/admin/shared/_tabnav_users.html.haml:29]: Can you use the translation file here?
```suggestion
  = tab t(".user_docs"), facility_user_user_file_uploads_path(current_facility, @user), secondary_tab == "docs"
```

**Review by giladshanan — APPROVED** [review]
One small nitpick

**Review by giladshanan — APPROVED** [review]

**Comment by stevens101** [comment]
It looks like the documents table column headings don't align with the data in the rows. For example the "File" column is actually the Delete column. Might need another th tag.

**Pull request merged**